### ISSUE TITLE
Some issues regarding the docs for app overriding

### ISSUE THREAD

**Issue by kkomninos** [ellmetha/django-machina#55]
Hi! I am trying to override a class in models.py following doc instructions. It finally worked but I would like to provide some points.

In overriding code example at https://django-machina.readthedocs.io/en/stable/customization/recipes/overriding_models.html :
```python 
# -*- coding: utf-8 -*-

from __future__ import unicode_literals

from machina.apps.forum_conversation.abstract_models import AbstractTopic

# Custom models should be declared before importing
# django-machina models

class Topic(AbstractTopic):
    icon = models.ImageField(verbose_name="Icon", upload_to="forum/topic_icons")

from machina.apps.forum_conversation.models import *  # noqa
```

seems to be missing:
```python
from django.db import models
```

In the whole example doc about overriding app/model, the app to override is reffered as "forum_conversation" but at https://django-machina.readthedocs.io/en/stable/customization/recipes/overriding_models.html the migration command is: `$ django-admin makemigrations forum_conversations`

Another issue I got:
'ImportError: No module named apps.forum_conversation.registry_config'
when apps directory was inside project directory.

I had to remove "yourproject" from line ```] + get_machina_apps(['yourproject.apps.forum_conversation', ])``` and move apps one folder up (out of project), or keep apps directory inside project and add project name to `registry_config` : `name = yourproject.apps.forum_conversation`

and `__init__.py`: `default_app_config = 'yourproject.apps.forum_conversation.registry_config.ConversationRegistryConfig'`

**Comment by ellmetha** [comment]
Hey @kkomninos! Thanks for your observations! Would you mind creating a Pull Request in order to fix these issues? This would be much appreciated. :smiley:

**Comment by kkomninos** [comment]
Yes I'll try to do it :)

**Comment by ellmetha** [comment]
Fixed in  69b8a99. Thanks! :wink:

**Comment by kkomninos** [comment]
Welcome :wink:
